### DISCUSSION TITLE
BTTV in Microsoft Edge Sidebar App not showing

### DISCUSSION THREAD

**Discussion by jentroverted** [night/betterttv#6808] [Q&A] [answered]
I've installed the BTTV extension for Edge, but when applying the Twitch app to the Edge Sidebar as a pinned app, BTTV does not show inside Twitch. Is this option not available inside the sidebar app for Edge? If not, any recommendations for 3rd partes to get BTTV in the app?

**Comment by night** [comment]
I'm not super familiar with the sidebar. Are you able to elaborate or provide a screenshot/video showing what you're referring to?

**Reply by jentroverted** [reply]
Hi there, this is a screenshot of the twitch app inside the sidebar. BTTV isn't showing even though the extension is enabled for Edge. 
![Screenshot 2024-05-15 115052](https://github.com/night/betterttv/assets/169923215/44f0ef0d-170a-4c93-ba1f-b53ae9b7d376)

**Reply by night** [reply, marked as answer]
It looks like this is a limitation of Edge's sidebar. No page extensions can work in the sidebar, including Adblockers: https://www.reddit.com/r/edge/comments/11m0yr0/make_extensions_work_on_the_sidebar/

**Reply by jentroverted** [reply]
yeah i figured that might be the case. Darn. I really like the audible notifications when someone responds to me.
